BETSE 0.9.2.0 bumped.

leycec opened the pull request:
This commit bumps conda-forge hosting to the most recent stable release: **BETSEE 0.9.2.0** (_Luckiest Levin_). The PyPI-hosted domain of this package's tarball has been updated from the now seemingly dead pypi.io to pypi.org. This is what redirects are for, infrastructure maintainers. `</sigh>`

@conda-forge-admin, please rerender

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

leycec committed:
BETSE 0.9.2.0 bumped.

This commit bumps conda-forge hosting to the most recent stable release:
BETSEE 0.9.2.0 (Luckiest Levin). The PyPI-hosted domain of this
package's tarball has been updated from the now seemingly dead pypi.io
to pypi.org. This is what redirects are for, infrastructure maintainers.
*sigh*

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leycec commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.7, conda-smithy 3.2.9, and conda-forge-pinning 2019.01.21

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leycec committed:
CircleCI rerun.

leycec commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

leycec commented:
**Urgh.** CircleCI now [spontaneously hiccups up](https://circleci.com/gh/conda-forge/betsee-feedstock/12?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) the [infamous `libGL.so.1` import error](https://github.com/conda-forge/pygridgen-feedstock/issues/10):

```
+ betsee --help
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/betsee_1548409688303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/betsee", line 10, in <module>
    sys.exit(main())
  File "/home/conda/feedstock_root/build_artifacts/betsee_1548409688303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/betsee/__main__.py", line 93, in main
    from betsee.guimetaapp import BetseeMetaApp
  File "/home/conda/feedstock_root/build_artifacts/betsee_1548409688303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/betsee/guimetaapp.py", line 24, in <module>
    from betsee.lib.pyside2.cache.guipsdcache import CachePolicy
  File "/home/conda/feedstock_root/build_artifacts/betsee_1548409688303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/betsee/lib/pyside2/cache/guipsdcache.py", line 23, in <module>
    from betsee.gui.simconf.stack.widget.guisimconfradiobtn import (
  File "/home/conda/feedstock_root/build_artifacts/betsee_1548409688303/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/betsee/gui/simconf/stack/widget/guisimconfradiobtn.py", line 21, in <module>
    from PySide2.QtWidgets import QButtonGroup, QRadioButton
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
Tests failed for betsee-0.9.2.0-py_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

If I had fine-grained control over the CircleCI configuration deployed here (...*do I?*), I'd just manually install the `libgl1` system library (e.g., with `apt install libgl1-mesa-glx` from an Anaconda docker image). I note the relevant `.circleci` and `.ci_support` dot directories, of course, but suspect that any desperate hacks I stuff into those directories would simply be overwritten by the next automated conda-forge rerendering run. Is there a sane(r) way to go about this?

@mingwandroid and @CJ-Wright: _**I summon thee!**_ You probably have more fulfilling quests to dispatch on a wintry Friday's eve, but the default CircleCI configuration for Qt-based conda-forge feedstocks appears to have gone badly awry. Although I know nothing, a recently misconfigured docker image might be the culprit.

Apologies for bothering you both so late into a bone-chilling January TGIF. May the underlying bug(s) be trivially dispatched and your weekends be merry and bright.

CJ-Wright commented:
I think you'll find this thread useful, or at least informative. 
https://gitter.im/conda-forge/conda-forge.github.io?at=5c409b109bfa375aab171e9b

leycec commented:
**Awesome.** Thanks for the heads up! I should have thought to check the gitter first. Because I am dense and only slightly inebriated, I'm still unclear what needs doing here. Maybe the PySide2 feedstock needs a `yum_requirements.txt` explicitly listing `libgl1` as a test-time requirement?

Let me pore through @mingwandroid's wizardly responses more deeply. :thinking:

leycec commented:
**Ah.** *Jetz ist alles klar.* @isuruf correctly identified the core issue – which, as expected, was an accidental docker misstep:

> We didn't have this problem because `linux-anvil` had `mesa-libGL` installed, but `linux-anvil-comp7` doesn't

..and @mariusvniekerk suggested the obvious solution – which, as expected, was amending the current dockerfile to install `mesa-libGL`:

> probably worthwhile to update the docker images to just have that in?

![Excellent.](https://media.giphy.com/media/kUFlw7XaGE36w/giphy.gif)

leycec commented:
**Addendum:** a relevant year-old issue already exists at ["add libgl to Anaconda docker images."](https://github.com/ContinuumIO/docker-images/issues/49) Of course, the last comment on that issue was... *nearly a year ago.* So this is why we can't have good things. :face_with_head_bandage:

leycec commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

leycec commented:
This will probably simply fail again, but let's force another CircleCI rerun. If this *does* fail again, we'll have little choice but to arthritically shake our fist at the putrescent winter sky and:

1. Temporarily disable all tests.
1. Accept this pull request.
1. Wait until packages are built.
1. Submit a new pull request re-enabling tests.

*y u do this 2 me, conda-forge.*

leycec committed:
CircleCI rerun (final).

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leycec commented:
**Very well.** You have chosen the bazooka, CircleCI. :boom:

leycec committed:
Functional tests disabled for CircleCI compliance.

These tests *MUST* be temporarily disabled until the docker image used
by CircleCI is revised to include the mandatory "mesa-libGL" dependency
installing the "libGL.so.1" shared library required by Qt at runtime.

leycec commented:
Let's see if we remember to re-enable functional tests in several months.

Since this glass is clearly half-empty, my gambling money is on _"Nope."_
